### PR TITLE
Prepare v8 for release

### PR DESCRIPTION
**Pull request by jakubfijalkowski**
- Rename `MapRemoteCqrs` to `MapRemoteCQRS`
- Update packages
- Adjust code to changes in the libraries
- Fix analyzer problems
- Add v8 changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+* Remove `AppContext` in favor of using `HttpContext` directly
+* Support Azure Workload Identity
+* Improve CoreLib-provided logging, tracing and metrics
+* Remove singleton/scoped FluentValidation configuration, remove `RuleForAsync` in favor of `CustomAsync`
+* Introduce exception translation middleware,
+* Add Ulids as a valid source-generated ID
+* Switch to `System.TimeProvider`, removing `ITimeProvider`
+* Improved PostgreSQL handling
+* UserId is now a generic type in most of the helpers
+* `IIdentifiable` is not `IEntity`
```
Comment (Dragemil) [inline CHANGELOG.md]: Let's be more straightforward with that one:
```suggestion
* Rename `IIdentifiable` to `IEntity`
```

File: test/LeanCode.TestCoverage.Tests/TestCoverageTests.cs [hunk 2]
```diff
@@ -1,9 +1,7 @@
-using Xunit;
-
 namespace LeanCode.TestCoverage.Tests;
 
 public class TestCoverageTests
 {
-    [Fact]
+    [Xunit.Fact]
```
Comment (Dragemil) [inline test/LeanCode.TestCoverage.Tests/TestCoverageTests.cs:5]: Why this change?

**Review by Dragemil — APPROVED** [review]

File: test/LeanCode.TestCoverage.Tests/TestCoverageTests.cs (hunk 2 above)
Comment (jakubfijalkowski) [reply test/LeanCode.TestCoverage.Tests/TestCoverageTests.cs:5]: Default rulesets of StyleCop (this is the only project where it is included) favor namespaced `using`s. This was the easiest fix.

File: CHANGELOG.md (hunk 1 above)
Comment (jakubfijalkowski) [reply CHANGELOG.md]: I could swear that this was a sentence in the original form. Will fix!
